packages/tools/devtools/devtools-example/src/App.tsx

```diff
@@ -93,6 +111,30 @@ async function populateRootMap(container: IFluidContainer): Promise<void> {
 	const sharedCounter = await container.create(SharedCounter);
 	rootMap.set(sharedCounterKey, sharedCounter.handle);
 
+	// Set up SharedTree for visualization
+	const sharedTreeFactoryObject = SharedTreeObject as unknown as SharedObjectClass<ISharedTree>;
+	const sharedTree = await container.create(sharedTreeFactoryObject);
+
+	// Populate SharedTree using Schema
+	const builder = new SchemaBuilder("SchemaAware");
+	const stringSchema = builder.primitive("string", ValueSchema.String);
```

Comment:
Let's check how `primitive()` works, what that first argument is, and if something like `stringProp`  might be clearer than just ` string` .